### PR TITLE
Add check for syntaxes files

### PR DESCRIPTION
**Pull request by ssbarnea**
- ensure syntaxes are built and in sync

### CODE REVIEW

File: scripts/build-syntaxes [hunk 1]
```diff
@@ -0,0 +1,9 @@
+#!/bin/bash
+set -euo pipefail
+
+for SRC in syntaxes/ansible/*.plist; do
+
+    DST=$(basename "$(basename "$SRC" .plist)")
```
Comment (webknjaz) [inline scripts/build-syntaxes:6]: could you rewrite this in Python?

File: package.json [hunk 2]
```diff
@@ -70,7 +70,7 @@
   "scripts": {
     "clean": "rimraf out/server && rimraf lib",
     "compile": "tsc -p .",
-    "lint": "npm ci && pre-commit run -a",
+    "lint": "npm ci && ./scripts/build-syntaxes && pre-commit run -a",
```
Comment (webknjaz) [inline package.json:73]: Why is this necessary before pre-commit?

Comment (ssbarnea) [reply package.json:73]: pre-commit will fail it if files are touched, including a nice diff. In the future I would rewrite it as pure python and make it a local hook but I do not have the energy to do it. It seems every minute I look over that repo I find other things that have zero CI testing.

Comment (webknjaz) [reply package.json:73]: Got it, thanks!

File: scripts/build-syntaxes (hunk 1 above)
Comment (webknjaz) [reply scripts/build-syntaxes:6]: Marking as resolved per comments in another thread.

**Review by webknjaz — APPROVED** [review]
